CMakeLists.txt

```diff
@@ -69,6 +69,13 @@ endif()
 set(ENABLE_UPDATE_CHECKER (NOT ${DEVELOPMENT_BUILD}) CACHE BOOL
 	"Whether to enable update checks by default")
 
+find_program(CCACHE_FOUND ccache)
+if (CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
```

Comment:
Please set `CMAKE_CXX_COMPILER_LAUNCHER` and `CMAKE_C_COMPILER_LAUNCHER` instead, as suggested by the cmake docs: https://cmake.org/cmake/help/latest/prop_gbl/RULE_LAUNCH_COMPILE.html

Reply:
fix it